server/util/body-parser.js

```diff
@@ -127,6 +128,20 @@ export function convertWpList(node) {
   }
 }
 
+function convertTweet(node) {
+  const className = node.attrs && getAttrVal(node.attrs, 'class');
+  const isTweet = Boolean(className && className.match('embed-twitter'));
+
+  if (!isTweet) return node;
+
+  return new BodyPart({
+    type: 'tweet',
+    value: new Tweet({
+      html: serializeNode(node)
+    })
```

Comment:
🚲 ⛺️ ing this - but it might be nice to remove the script tag, and just add it once.
It doesn't make that much of a difference as it's async and would be heavily cached.